[2024-09-08T19:11:30.285+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-08T19:11:30.296+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sum_dag_with_taskflow_api.get_sum scheduled__2024-09-05T02:00:00+00:00 [queued]>
[2024-09-08T19:11:30.304+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sum_dag_with_taskflow_api.get_sum scheduled__2024-09-05T02:00:00+00:00 [queued]>
[2024-09-08T19:11:30.305+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 4
[2024-09-08T19:11:30.314+0000] {taskinstance.py:2879} INFO - Executing <Task(_PythonDecoratedOperator): get_sum> on 2024-09-05 02:00:00+00:00
[2024-09-08T19:11:30.322+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=11103) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-08T19:11:30.325+0000] {standard_task_runner.py:72} INFO - Started process 11108 to run task
[2024-09-08T19:11:30.323+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'sum_dag_with_taskflow_api', 'get_sum', 'scheduled__2024-09-05T02:00:00+00:00', '--job-id', '276', '--raw', '--subdir', 'DAGS_FOLDER/test_dag.py', '--cfg-path', '/tmp/tmpdsprkzd0']
[2024-09-08T19:11:30.327+0000] {standard_task_runner.py:105} INFO - Job 276: Subtask get_sum
[2024-09-08T19:11:30.360+0000] {task_command.py:467} INFO - Running <TaskInstance: sum_dag_with_taskflow_api.get_sum scheduled__2024-09-05T02:00:00+00:00 [running]> on host 626f2e28dec2
[2024-09-08T19:11:30.454+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Abhay' AIRFLOW_CTX_DAG_ID='sum_dag_with_taskflow_api' AIRFLOW_CTX_TASK_ID='get_sum' AIRFLOW_CTX_EXECUTION_DATE='2024-09-05T02:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-05T02:00:00+00:00'
[2024-09-08T19:11:30.456+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-08T19:11:30.459+0000] {logging_mixin.py:190} INFO - sum of 0 & 0 is 0
[2024-09-08T19:11:30.460+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-08T19:11:30.468+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-08T19:11:30.469+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=sum_dag_with_taskflow_api, task_id=get_sum, run_id=scheduled__2024-09-05T02:00:00+00:00, execution_date=20240905T020000, start_date=20240908T191130, end_date=20240908T191130
[2024-09-08T19:11:30.502+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-09-08T19:11:30.515+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-08T19:11:30.516+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
